backend/manager/modules/common/src/main/java/org/ovirt/engine/core/common/businessentities/storage/TransferClientType.java

```diff
@@ -31,11 +31,6 @@ public static TransferClientType forValue(int value) {
         return valueToType.get(value);
     }
 
-    @Override
-    public String toString() {
-        return description;
```

Comment:
without this, there's no need for the `description` field, no? I see it (the description) is used in ImageTransfer#toString - if that's the only place (I could have missed other places, it was a quick search) then I think it's ok but we better remove it then